src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
-});
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+wall.addEventListener('click', moveSpider);
+
+function moveSpider(event) {
+  const spiderMaxCoordX = this.clientWidth - spider.clientWidth;
```

Comment:
it's better to use spider.offsetWidth (spider's full size)

Reply:
done